pkgs/development/libraries/agda/agda-prelude/default.nix

```
@@ -1,23 +1,32 @@
-{ stdenv, agda, fetchgit }:
+{ stdenv, mkDerivation, fetchFromGitHub }:
 
-agda.mkDerivation (self: rec {
-  version = "eacc961c2c312b7443109a7872f99d55557df317";
-  name = "agda-prelude-${version}";
+mkDerivation rec {
+  version = "compat-2.6.0";
+  pname = "agda-prelude";
```

```suggestion
  pname = "prelude";
```

(And rename the folder accordingly)

---

Same as above, this is the name used in the agda-lib file